win32ss/user/ntuser/hook.c

```diff
@@ -25,6 +25,15 @@ PPROCESSINFO ppiUahServer;
 
 /* PRIVATE FUNCTIONS *********************************************************/
 
+BOOL FASTCALL UserIsItHooked(_In_ INT HookId)
+{
+    PTHREADINFO pti = (PTHREADINFO)PsGetCurrentThreadWin32Thread();
+    const INT nFlag = HOOKID_TO_FLAG(HookId);
```

Comment:
`UINT` instead.
`fsHooks` member is unsigned as well.

Reply:
```suggestion
    const UINT nFlag = HOOKID_TO_FLAG(HookId);
```